cli/commands/init.go

```diff
@@ -102,38 +105,43 @@ func (o *InitOptions) Run() error {
 			return handlePromptError(err)
 		}
 		if strings.TrimSpace(r) != "" {
-			dir = r
+			o.InitDir = r
 		} else {
-			dir = defaultDirectory
+			o.InitDir = defaultDirectory
 		}
-	} else {
-		dir = o.InitDir
 	}
-
-	if o.EC.ExecutionDirectory == "" {
+	if o.InitDir == "." {
```

Comment:
@ShraddhaAg a nitpick, 
Instead of directly comparing to `.` do you think something similar to the following is better?
```
	cwdir, err := os.Getwd()
	if err != nil {
		return errors.Wrap(err, "error getting current working directory")
	}
	initPath, err := filepath.Abs(o.InitDir)
	if err != nil {
		return err
	}
	if initPath == cwdir {
             // do stuff
        }
```
This can also handle paths like `./`

Reply:
Makes sense. Updated accordingly :smile: